### DISCUSSION TITLE
CF Variable types and rdf usage

### DISCUSSION THREAD

**Discussion by cpina** [frictionlessdata/frictionlessdata.io#599] [Q&A]
In this tabular data resource:
https://github.com/proccaserra/rose2018ng-notebook/blob/master/data/processed/rose-data/rose-aroma-data-integration-datapackage.json#L64

it uses `rdfType`.

There have been discussions to have `units` in the tabular data resource (see https://github.com/frictionlessdata/specs/issues/537 or https://github.com/frictionlessdata/specs/issues/216). Related to units there are "variable types" to indicate what a variable represents even when the units are the same. For example: "air temperature in celsius" and "sea temperature in celsius".

There is a standard way to describe them using the "Climate Forecast convention": http://cfconventions.org/ . Here there is the list of variable types: http://cfconventions.org/Data/cf-standard-names/72/build/cf-standard-name-table.html

I haven't used semantic web, rdf types, etc. My question is: could anyone use `rdfType` in a tabular-schema to say "this column is of the class sea temperature": http://cfconventions.org/Data/cf-standard-names/72/build/cf-standard-name-table.html#sea_surface_temperature ?

The Frictionless Data spec says: https://specs.frictionlessdata.io/table-schema/#types-and-formats

`
The value of of the rdfType property MUST be the URI of a RDF Class, that is an instance or subclass of RDF Schema Class object
`

Could you give me some pointers on RDF Class, etc.? I see that https://schema.org/Country has things like:
`  <div id="mainContent" vocab="http://schema.org/" typeof="rdfs:Class" resource="http://schema.org/Country">`

But http://edamontology.org/format_1197 doesn't have the same structure.

Should the linked site (http://cfconventions.org/Data/cf-standard-names/72/build/cf-standard-name-table.html#sea_surface_temperature) have anything specific in order to be used as `rdfType`? I'd like to understand a bit more, from the Frictionless Data point of view, how `rdfType` should be used, what the linked website should have (`rdfs:` tags in the HTML?)

Please, feel free to send me to more documentation, examples, validators, etc.

cfconventions uses github https://github.com/cf-convention . If I understood what needs to be done on their side I'd be happy to at least explain the use case of Frictionless Data to them in case that they could adapt part of their code.

**Comment by cpina** [comment]
@lwinfree you might be interested in this as well

**Comment by lwinfree** [comment]
copying in @proccaserra for related interest
